### ISSUE TITLE
manager.lua invalid type boolean, expected uuid during /fleetcontrol enable and disable

### ISSUE THREAD

**Issue by mccorkle** [w00zla/avorion-fleetcontrol#2]
## Issue
when attempting to enable fleetcontrol ui, I get no Fleet Control icon added to my UI.

## Debug output
I saw the error was in manager, so I placed the following line on [#142](https://github.com/w00zla/avorion-fleetcontrol/blob/master/scripts/player/fleetcontrol/manager.lua#L142):
`debugLog("FOO %s %s", shipidx, lastCraft)`

Based on that, it looks like lastCraft is nil

```
SCRIPT FleetControl [v0.4] p#1 DEBUG => FOO 249b6069-89da-4dBa-84e8-c36b0c510089 nil
could not execute function 'validateClientVersion' in 'data/scripts/player/fleetcontrol/manager.lua':
invalid type boolean at stack position 1, expected 'Uuid'
stack traceback:
  [C]-1: in function __lt
  data/scripts/player/fleetcontrol/manager:142 in function addShipUIScript
  data/scripts/player/fleetcontrol/manager:89 in function initShitUIHandling
  data/scripts/player/fleetcontrol/manager.lua:68: in function ?
setting state to invalid.
```
NOTE: In the hand typed stack backtrace, I fudged the line number to be correct, even though I added my debugLog to the line before, so my screenshot has this error in lines 144 and 143.  
![avorian fleet command error](https://user-images.githubusercontent.com/2091058/28596680-fb883c50-7167-11e7-8b00-2ee73695ffa3.jpg)

## How to reproduce
* Download v0.4 of fleet control from github
* install it into \steamapps\common\Avorion\data
* Load game locally (have not tested on server)
* bring up chat with <enter>
* `/fleetcontrol enable true`

## Expected Outcome
Expected to have a much better ui to manage my mining fleet.

## Versions:
* Avorian Client 0.12.7 r8782
* FleetCommand v0.4

**Comment by VelliusGit** [comment]
Functions **addShipUIScript** and **removeShipUIScript** in **data/scripts/player/fleetcontrol/manager.lua** does a math check against **shipidx** which is a **UUID** (ex: 43214321231-312312-312321)

Removing the `shipidx > 0` fix the issue.

**Comment by nkv123** [comment]
same problem, different message in console

![avorion_001](https://user-images.githubusercontent.com/4409902/29102352-c9c5d0cc-7cb8-11e7-96d3-56318630e67d.png)

Expected Outcome

Icon to appear and mod to work


Versions:

Avorian Client 0.12.7 r8782
FleetCommand v0.4

**Comment by w00zla** [comment]
Fixed in [commit b458bbe41e8f8e923785cc80e70310997d949e54](https://github.com/w00zla/avorion-fleetcontrol/commit/b458bbe41e8f8e923785cc80e70310997d949e54)!
